Description: Assure that headers from libNX_X11 are always included from ${INCDIR}/nx-X11/.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

Index: Mesa/include/GL/xmesa.h
===================================================================
--- Mesa.orig/include/GL/xmesa.h
+++ Mesa/include/GL/xmesa.h
@@ -75,8 +75,8 @@ extern "C" {
 #ifdef XFree86Server
 #include "xmesa_xf86.h"
 #else
-#include <X11/Xlib.h>
-#include <X11/Xutil.h>
+#include <nx-X11/Xlib.h>
+#include <nx-X11/Xutil.h>
 #include "xmesa_x.h"
 #endif
 #include "GL/gl.h"
Index: Mesa/src/mesa/drivers/dri/common/glcontextmodes.c
===================================================================
--- Mesa.orig/src/mesa/drivers/dri/common/glcontextmodes.c
+++ Mesa/src/mesa/drivers/dri/common/glcontextmodes.c
@@ -39,7 +39,7 @@
 # include "imports.h"
 # define __glXMemset  memset
 #else
-# include <X11/X.h>
+# include <nx-X11/X.h>
 # include <GL/glx.h>
 # include "GL/glxint.h"
 
@@ -49,7 +49,7 @@
 #  define _mesa_malloc(b) malloc(b)
 #  define _mesa_free(m)   free(m)
 # else
-#  include <X11/Xlibint.h>
+#  include <nx-X11/Xlibint.h>
 #  define __glXMemset  memset
 #  define _mesa_malloc(b) malloc(b)
 #  define _mesa_free(m) free(m)
Index: Mesa/src/mesa/drivers/x11/glxheader.h
===================================================================
--- Mesa.orig/src/mesa/drivers/x11/glxheader.h
+++ Mesa/src/mesa/drivers/x11/glxheader.h
@@ -43,8 +43,8 @@
 
 #else
 
-# include <X11/Xlib.h>
-# include <X11/Xutil.h>
+# include <nx-X11/Xlib.h>
+# include <nx-X11/Xutil.h>
 # ifdef USE_XSHM  /* was SHM */
 #  include <sys/ipc.h>
 #  include <sys/shm.h>
Index: Mesa/src/mesa/glapi/glthread.h
===================================================================
--- Mesa.orig/src/mesa/glapi/glthread.h
+++ Mesa/src/mesa/glapi/glthread.h
@@ -180,7 +180,7 @@ typedef CRITICAL_SECTION _glthread_Mutex
  * We wrap it again for GL.
  */
 #ifdef USE_XTHREADS
-#include <X11/Xthreads.h>
+#include <nx-X11/Xthreads.h>
 
 typedef struct {
    xthread_key_t key;
